tmt/steps/provision/testcloud.py

```diff
@@ -531,6 +541,79 @@ def _generate_disk_filepaths() -> Iterator[Path]:
             )
 
 
+def _apply_cpu_processors(
+        hardware: Optional[tmt.hardware.Hardware],
+        domain: 'DomainConfiguration',
+        logger: tmt.log.Logger) -> None:
+    """ Apply ``tpm`` constraint to given VM domain """
```

Comment:
Should be cpu.processors instead of tpm here ?

Reply:
Correct, fixed in f72c0081